lib/local-links-manager/import/links.rb

```diff
@@ -0,0 +1,39 @@
+require "csv"
+
+module LocalLinksManager
```

Comment:
You can do `class LocalLinksManager::Import::Links` here to avoid the indentation.

Reply:
You can, but that relies on the modules having been defined first elsewhere so it's usual (though visually annoying) to be explicit about it in each file.

We haven't specified an opinion via linting yet..
https://github.com/alphagov/govuk-lint/blob/3bc900f0b5c30541dd3d467f341a2263728a3ce2/configs/rubocop/other-style.yml#L73-L75

Reply:
Hmm interesting, I've always used the compacted style and haven't had problems with relying on modules having been defined already, but I can see why that might be problematic. I don't really mind either way.

Reply:
It might be worth testing out and revisiting so we can settle on one or the other.  I'm happy to use the compact style provided it works consistently and doesn't rely on the order of files being loaded.

Reply:
Discussed offline: I'll leave it expanded in this PR, and we can revisit once we have an agreed and consistent preferred style.